script/Arcueid/Day 10/10_00_ARC10_5.txt

```diff
@@ -565,10 +564,7 @@ But is it really so bad to do something pointless?"
 [sha:a113c80268002f18148192bc606874d0601e01b7]{
 -- Page 435, Offset 25369. Mods: @n.
 -- 「いいじゃないか、余分な事って
-// 余分 kiiinda touching on a theme here? Using same word here as 25392.
-// Note no period, is glued
-"There's no harm in it, right?
- Doing unnecessary things, I mean
+"It's fine, right? To waste time, that is
```

Comment:
revert, doing unnecessary things is an important thing for arc's arc. (a perfect machine that broke down)